php-classes/Google/DriveFile.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Google;
+
+use \Slate\CBL\Tasks\Attachments\GoogleDriveFile;
+use \Slate\CBL\Tasks\Task;
+use \Slate\CBL\Tasks\StudentTask;
```

Comment:
@nbey code in the `Google` namespace should not be relying on code in the `Slate` namespace. Any `Slate`-dependent code should be in the `Slate` namespace. It doesn't look like any of these are actually used in this file anyway.

Also, `use` statements don't need to start with `\`, they are inherently absolute.